packages/grafana-ui/src/components/TimePicker/TimeOfDayPicker.tsx

```diff
@@ -16,6 +16,11 @@ interface Props {
 }
 
 const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  const bgColor = stv({ light: theme.colors.white, dark: theme.colors.gray15 }, theme.type);
```

Comment:
should this not be formInputBg

Reply:
Ah right, didn't know `gray15` had an alias.